develop/interoperability/send-messages.md

```diff
@@ -0,0 +1,44 @@
+---
+title: XCM - Send Messages
+description: Unlock blockchain interoperability with XCM — Polkadot's Cross-Consensus Messaging format for cross-chain interactions.
+---
+
+# XCM - Send Messages
+
+## XCM Frame Pallet Overview
+
+The XCM pallet [(pallet-xcm)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/pallet-xcm/src/lib.rs) provides a set of pre-defined, commonly used XCVM programs in the form of a set of extrinsics using [FRAME](https://docs.substrate.io/reference/frame-pallets/).
+
+This pallet provides some default implementations for traits required by XcmConfig. The XCM executor is also included as an associated type within the pallet's configuration. For further details on XCM Configurations, see the previous page here.
```

Comment:
Some feedback here:

- I'm checking the docs of `pallet-xcm` and I struggling to find this: "provides some default implementations for traits". Are you referring to https://paritytech.github.io/polkadot-sdk/master/pallet_xcm/index.html#functions ?

- I think we should add a reference to `XcmConfig` coming from polkadot sdk rust docs

- XCM executor should be linked to https://paritytech.github.io/polkadot-sdk/master/staging_xcm_executor/struct.XcmExecutor.html